in_kafka: Add async_commit description (#1385)
fluent/fluent-bit-docs: pipeline/inputs/kafka.md

| poll\_ms | Kafka brokers polling interval in milliseconds. | 500 |
| Buffer\_Max\_Size | Specify the maximum size of buffer per cycle to poll kafka messages from subscribed topics. To increase throughput, specify larger size. | 4M |
| poll\_ms | Kafka brokers polling interval in milliseconds. | 500 |

> | async_commit | Enable Kafka commit operation asynchronously. This option takes a boolean value: True/False. | False |

Reviewer: ```suggestion
| async_commit | Enable Kafka commit operation asynchronously. This parameter accepts a boolean value of `true` or `false`. | `false` |
```

Author: @lockewritesdocs 
Will incorporate your suggestion. Thanks for the review👍
